src/ContentElement/NodesContentElement.php

```diff
@@ -96,8 +96,7 @@ public static function generateBackendWildcard(array $data, array $ids): string
             }
         }
 
-		$type = new UnicodeString($GLOBALS['TL_LANG']['FMD'][$data['type']][0]);
-        $wildcard = '### '.$type->upper().' ###';
+        $wildcard = '### '.(new UnicodeString($GLOBALS['TL_LANG']['FMD'][$data['type']][0]))->upper().' ###';
```

Comment:
How about just changing this to:
mb_strtoupper($GLOBALS['TL_LANG']['FMD'][$data['type']][0], 'UTF8) and not going for the requirement of symfony/string at all.

That way it will still be uppercase, work for contao 4.4 and even 4.13 with the `text-transform: uppercase` as it doesn't matter.